platforms/documentation/docs/src/docs/userguide/authoring-builds/plugins/reporting_problems.adoc

```diff
@@ -95,3 +95,26 @@ The progress listener callback should then check if the operation result is of t
 
 In addition, there's a more convenient way to access the failure details.
 If clients configure the project connection with `LongRunningOperation.withFailureDetails()`, the Tooling API implicitly subscribes to the `ROOT` operation type and provides failure details via the `GradleConnectionException.getFailures()` method.
+
+[[sec:generated_html_report]]
+=== Generated HTML report
```

Comment:
@donat WDYT?

Reply:
I'm happy with the content, but the screenshot is not right. It is only a demonstration for the problem summarisation.

You can probably find a better example amongst the Gradle CI builds.